python_modules/dagster/dagster/core/execution_plan/objects.py

```diff
@@ -49,7 +48,17 @@ def value_repr(self):
         return self._value_repr
 
     def get_value(self):
-        return self._intermediates_manager.get_value(self.step_output_handle)
+        # FIXME:
+        # For now we are disallowing getting the value for anything
+        # except the in-memory version of this. get_value will need to put
+        # on higher level object that will have access to pipeline_context
+
+        from dagster.core.intermediates_manager import InMemoryIntermediatesManager
```

Comment:
is there a strong reason to bury this import here

Reply:
nope! thanks for good catch